wandb/__init__.py

```diff
@@ -714,4 +748,8 @@ def set_global_config(run):
     return run
 
 
-__all__ = ['init', 'config', 'termlog', 'run', 'types', 'callbacks', 'join']
+tensorflow = util.LazyLoader('tensorflow', globals(), 'wandb.tensorflow')
+tensorboard = util.LazyLoader('tensorboard', globals(), 'wandb.tensorboard')
```

Comment:
A new lazy loader to ensure we don't pull in tensorflow unless someone imports or accesses `wandb.tensorflow`